docs/source/virtual_ecosystem/implementation/abiotic_implementation.md

```diff
@@ -58,45 +58,158 @@ display_markdown(
+The **aerodynamic resistance of the soil surface** is given by:
 
-$$r_{A} = \frac {C_{S}}{u_{b}}$$
+$$r_{A} = \frac {C_{E}}{u}$$
 
-where $u_{b}$ is the wind speed in the bottom air layer and $C_{S}$ is
-the soil surface heat transfer coefficient.
+where:
 
-Latent heat flux $\lambda E_S$ is derived by conversion of surface evaporation as
-calculated by the hydrology model, and ground heat flux $G$ is calculated as the residual:
+$u$:
+Horizontal wind speed at the bottom air layer ($\mathrm{m\,s^{-1}}$)
+
+$C_E$:
+Drag coefficient for evaporation (–)
+
+The **latent heat flux** is derived by conversion of surface evaporation as
+calculated by the hydrology model.
+
+The **ground heat flux** $G$ is calculated as the residual of the energy balance:
 
 $$G = R_N - H_S - \lambda E_S$$
 
-After the flux partitioning, we determine the soil temperatures at different depths.
-At the moment, this is achieved with linear interpolation between the surface and
-soil temperature at 1 m depth. In the future, we aim for a mechanistic implementation.
+### Soil temperature update
+
+After the energy fluxes at the land surface have been partitioned, we simulate how heat
+is transported vertically through the soil profile by updating the temperature of each
+soil layer over time. This is done using an explicit finite-difference approach, which
+numerically solves the one-dimensional heat diffusion equation. The method accounts for
+thermal diffusivity and the net ground heat flux to calculate temperature changes at
+each soil depth.
+
+The **soil thermal diffusivity** $\alpha$ ($\mathrm{m^{2}\,s^{-1}}$) determines the rate
+at which heat is conducted through the soil. It is defined as:
+
+$$\alpha = \frac{\lambda}{\rho c}$$
+
+where:
+
+$\lambda$:
+Soil thermal conductivity ($\mathrm{W\,m^{-1}\,K^{-1}}$), indicating how
+  easily heat moves through soil
+
+$\rho$:
+Soil bulk density ($\mathrm{kg\,m^{-3}}$), including solids and pore spaces
+
+$c$:
+Soil specific heat capacity ($\mathrm{J\,kg^{-1}\,K^{-1}}$), the energy required to raise
+the temperature of 1 kg of soil by 1 K.
+
+#### Temperature Update Scheme
+
+Let $T_i^t$ represent the temperature (°C or K) of the $i^\text{th}$ soil layer at time
+$t$. The soil column is discretized into $n$ layers, each of thickness $\Delta z$ (m),
+and time advances in steps of $\Delta t$ (s).
+
+**Top layer update** (surface boundary condition):
+
+The topmost layer ($i = 0$) is updated using the net ground heat flux $G$
+($\mathrm{W\,m^{-2}}$):
+
+$$T_0^{t+\Delta t} = T_0^t + (\frac{\Delta t}{(\rho c \Delta z)}) G$$
+
+**Interior layers update**:
+
+Each interior layer ($i = 1, \dots, n-2$) exchanges heat with adjacent layers following
+the diffusion equation:
+
+```{math}
+\begin{aligned}
+T_i^{t+\Delta t} =
+& T_i^t + (\frac{\Delta t}{\Delta z^2}) \alpha (T_{i+1}^t - 2T_i^t + T_{i-1}^t)
+\end{aligned}
+```
+
+This term approximates vertical conduction using the second spatial derivative of
+temperature.
```

Comment:
Might be worth giving either a citation to or writing out the temperature function here